community/modules/scripts/spack-install/templates/install_spack.tpl

```diff
@@ -9,6 +9,11 @@ if [[ $EUID -ne 0 ]]; then
   exit 1
 fi
 
+# Activate ghpc-venv virtual environment if it exists
+if [ -d /usr/local/ghpc-venv ]; then
+  source /usr/local/ghpc-venv/bin/activate
```

Comment:
This is a sad side repercussion of having virtual env. Should we rather fix all scripts that expect ansible, or just install in the system?

Reply:
Yeah, I don't like this. 
Right now, I'm tempted to pull the plug on the virtual environment for now and put it back in later when I/we have a better understanding of the effects.

Reply:
You could do what @tpdownes had done before which is to still install into a virtual env but then symbolic link to the that virtual env, exposing it to the system: [example](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/8525bf230cbf67ae30d58295243c777ee82f7f24/modules/scripts/startup-script/examples/install_ansible.sh#L43).

I am also fine to do that as a follow on as we have people waiting for a newer version of ansible as of this moment.

Reply:
It is also possible to invoke any pip-installed binary with a full path. It will do the "right thing" in terms of Python libraries e.g. swap `/usr/bin/ansible-playbook` for `/usr/local/ghpc-venv/bin/ansible-playbook`.